xrootd/protocol/signRequirements.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
```

Comment:
by convention, we don't use very much camelCase names for files.
I would either rename `signRequirements.go` into `signing.go` or `signed_requirements.go`.

or, perhaps, just merge the content of that file at the bottom of `protocol.go` ?

Reply:
I would like not to put it into `protocol.go` because I expect a lot of "configuration" code to be placed here to initialize the predefined levels. I'm fine with `signing.go`. :)